### PR TITLE
Apply new PS rebranding to TopContributors

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | This change implements the [rebranding](https://build.prestashop-project.org/news/2023/new-project-branding/)
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no

### CODE REVIEW

File: assets/style/_header.scss [hunk 1]
```diff
@@ -1,28 +1,32 @@
 header {
-  height: 32.5rem;
+
+  h1 {
+    font-family: 'Prestafont';
```
Comment (boherm) [inline assets/style/_header.scss:4]:
```suggestion
    font-family: Prestafont, sans-serif;
```
And for other places too!

File: assets/style/_header.scss [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
 
   .gradient {
     width: 100%;
     height: 100%;
     background: $primary;
   }
 
-  .white-text {
+  .black-text {
+    font-family: 'IBM Plex Sans', sans-serif;
```
Comment (boherm) [inline assets/style/_header.scss:15]:
```suggestion
    font-family: 'IBM Plex Sans', Arial, sans-serif;
```
And same for all other places with IBM Plex Sans :D

**Review by boherm — CHANGES_REQUESTED** [review]
LGTM, anyway just few comments to be better with fonts!

File: assets/style/_header.scss (hunk 2 above)
Comment (matks) [reply assets/style/_header.scss]: Fixed in https://github.com/PrestaShop/TopContributors/pull/118/commits/ccb67fff2bd015ae2c21aa81fdc391f713eaf909

File: assets/style/_header.scss (hunk 1 above)
Comment (matks) [reply assets/style/_header.scss]: Fixed in https://github.com/PrestaShop/TopContributors/pull/118/commits/ccb67fff2bd015ae2c21aa81fdc391f713eaf909

**Pull request merged**